### PR TITLE
fix(plugin-meetings): using the env for port in the web pack config

### PR DESCRIPTION
**Pull request by Neeraj-swarnkar**
# COMPLETES #[ SPARK-303051 ](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-303051)

## This pull request addresses

[SPARK-303051](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-303051) Thinkrite - JS SDK - sample slow on PORT 8080 

## by making the following changes

Using the .env for PORT in the web pack config
Screnshot added below -

Screenshot 2022-06-20 at 7 35 48 PM

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ module.exports = (env = {NODE_ENV: process.env.NODE_ENV || 'production'}) => ({
     'eval-cheap-module-source-map' : 'source-map',
   devServer: {
     https: true,
-    port: 8000,
+    port: process.env.PORT || 8000,
```
Comment (Neeraj-swarnkar) [inline webpack.config.js:39]: Using the .env for PORT inside the web pack config file.

Comment (arun3528) [reply webpack.config.js:39]: looks good

Comment (Neeraj-swarnkar) [reply webpack.config.js:39]: @arun3528, Thanks..

**Review by mkesavan13 — APPROVED** [review]
Looks good to me too. Approving

**Pull request merged**